besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -569,6 +570,15 @@ void setBannedNodeIds(final List<String> values) {
           "Comma separated list of APIs to enable on JSON-RPC HTTP service (default: ${DEFAULT-VALUE})")
   private final List<String> rpcHttpApis = DEFAULT_RPC_APIS;
 
+  @Option(
+      names = {"--rpc-http-api-method-no-auth", "--rpc-http-api-methods-no-auth"},
```

Comment:
I think just having "--rpc-http-api-methods-no-auth" is enough. I find it confusing to have both here ...

Reply:
ah this is just following the way we did '--rpc-http-api-method' and '--rpc-http-api-methods', keeping it consistent